iceoryx_binding_c/source/c_wait_set.cpp

```diff
@@ -68,12 +68,10 @@ uint64_t iox_ws_timed_wait(iox_ws_t const self,
                            const uint64_t eventInfoArrayCapacity,
                            uint64_t* missedElements)
 {
-    return event_info_vector_to_c_array(
-        self->timedWait(units::Duration::fromNanoseconds(timeout.tv_nsec)
-                        + units::Duration::fromSeconds(timeout.tv_sec)),
-        eventInfoArray,
-        eventInfoArrayCapacity,
-        missedElements);
+    return event_info_vector_to_c_array(self->timedWait(units::Duration(timeout) + units::Duration(timeout)),
```

Comment:
oh, sorry. I meant a single `units::Duration(timeout)` without splitting the seconds and nanoseconds into two separate durations just to add them immediately afterwards. The `Duration` ctor can take a `timespec` as argument

Reply:
Dunno why I didn't noticed it :D Thanks.

Reply:
Okey, should be okey now. Thanks!